web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/LeggTilGrunnlagBosituasjonRoutes.kt

```diff
@@ -137,3 +148,43 @@ internal fun Route.leggTilGrunnlagBosituasjonRoutes(
         }
     }
 }
+
+internal fun SøknadsbehandlingService.KunneIkkeVilkårsvurdere.tilResultat(): Resultat {
+    return when (this) {
+        SøknadsbehandlingService.KunneIkkeVilkårsvurdere.FantIkkeBehandling -> fantIkkeBehandling
+        is SøknadsbehandlingService.KunneIkkeVilkårsvurdere.FeilVedValideringAvBehandlingsinformasjon -> this.feil.tilResultat()
+        SøknadsbehandlingService.KunneIkkeVilkårsvurdere.HarIkkeEktefelle -> harIkkeEktefelle
+    }
+}
+
+internal fun Søknadsbehandling.KunneIkkeOppdatereBosituasjon.tilResultat(): Resultat {
+    return when (this) {
+        is Søknadsbehandling.KunneIkkeOppdatereBosituasjon.KlarteIkkeHentePersonIPdl -> Feilresponser.fantIkkePerson
+        is Søknadsbehandling.KunneIkkeOppdatereBosituasjon.KunneIkkeLageGrunnlagsdata -> this.feil.tilResultat()
+        is Søknadsbehandling.KunneIkkeOppdatereBosituasjon.UgyldigTilstand -> ugyldigTilstand(this.fra, this.til)
+    }
+}
+
+internal fun KunneIkkeLageGrunnlagsdata.tilResultat(): Resultat {
+    return when (this) {
+        KunneIkkeLageGrunnlagsdata.FradragForEpsSomIkkeHarEPS -> HttpStatusCode.BadRequest.errorJson(
```

Comment:
Jeg hadde ikke forventet at vi kunne få feil relatert til fradrag når vi oppdaterer bosituasjon? Jeg ble litt usikker på hele den: Skal vi fjerne fradragene når man fjerner EPS? Er det ikke litt vanskeligere for brukeren dersom hen får en feilmelding om at han må fjerne alle EPS sine fradrag før han fjerner EPS?

Reply:
Nå er det jo kanskje slik at ikke alle feilmeldingene vil kunne skje? men pga hvordan dem er satt opp, så er det alltid 'en mulighet'